userappwithbroker/build.gradle

```diff
@@ -16,7 +16,7 @@ allprojects {
     }
 }
 
-def adalVersion = "3.1.2"
+def adalVersion = "3.1.3"
```

Comment:
intentional?

Reply:
Yes.  We moved ADAL versions, should be up to date.